cmd/minishift/cmd/start.go

```diff
@@ -405,6 +405,8 @@ func startHost(libMachineClient *libmachine.Client) *host.Host {
 		cacheMinishiftISO(machineConfig)
 
 		fmt.Print("-- Starting Minishift VM ...")
+	} else {
+		fmt.Print("-- Starting to Provision Remote Machine ...")
```

Comment:
Why the Caps for Each Word?

Reply:
suggestion: `Starting to provision the remote machine`.